DancingGoat/CultureMissingMiddleware.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DancingGoat
+{
+    public class CultureMissingMiddleware
+    {
+        private readonly string[] _cultures = { "en-US", "es-ES" };
+        private readonly RequestDelegate _next;
+
+        public CultureMissingMiddleware(RequestDelegate next)
+        {
+            _next = next;
+        }
+
+        public async Task InvokeAsync(HttpContext context)
+        {
+            var parameters = context.Request.Path.Value.Split("/");
```

Comment:
there are no comments regarding what this code does...what does it parse, what's the result